frontend/src/app/pages/content-management/content-management.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { Product } from 'src/app/interfaces/product';
+import { ProductService } from 'src/app/services/product/product.service';
+
+@Component({
+  selector: 'app-content-management',
+  templateUrl: './content-management.component.html',
+  styleUrls: ['./content-management.component.scss']
+})
+export class ContentManagementComponent implements OnInit {
+
+  public products: Product[];
+
+  constructor(
+    public productService: ProductService,
+  ) { }
+
+  ngOnInit(): void {
+    this.getProducts();
+  }
+
+  public getProducts(): void {
+    this.productService.getProducts().subscribe((products: Product[]) => this.products = products);
+  }
+
+  public saveProduct(product: Product): void {
```

Comment:
It is wrong naming "saveProduct". In development we have 4 base action(GET, CREATE, UPDATE, DELETE). These 4 action names CRUD. In future try to use these actions instead of "save" or "add"